src/plugins/objects/bank/bank.plugin.ts

```diff
@@ -170,7 +172,11 @@ export const withdrawItem: itemInteractionActionHandler = (details) => {
         amount: removeFromContainer(playerBank, details.itemId, countToRemove)
     };
 
-    playerInventory.add({ itemId: itemToAdd.itemId, amount: itemToAdd.amount });
+    if (stackable)
```

Comment:
Use brackets with your `if statements`, just for readability.

Reply:
👍 Wasn't sure what the projects stance was on this, may be worth adding to the linting rules though too.